docker/Dockerfile.xenial_debug

```diff
@@ -2,11 +2,14 @@ FROM ubuntu:xenial
 # Base image for debug builds.
 # Built manually uploaded as "istionightly/base_debug"
 
+ENV DEBIAN_FRONTEND=noninteractive
+
 # Do not add more stuff to this list that isn't small or critically useful.
 # If you occasionally need something on the container do
 # sudo apt-get update && apt-get whichever
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
+RUN apt-get update
```

Comment:
This image needs to be one layer to avoid bloat of the debug image size.  We have semi-made a decision we are not gong to use the `docker squash` feature, atleast not in the short term.